alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/apiClient.mustache

```diff
@@ -4,7 +4,7 @@ import org.springframework.cloud.openfeign.FeignClient;
 import {{configPackage}}.ClientConfiguration;
 
 {{=<% %>=}}
-@FeignClient(value = "alfresco<%classname%>", url = "${<%artifactId%>.url}", configuration = ClientConfiguration.class)
+@FeignClient(value = "<%vendorExtensions.x-service-url-prefix%>Alfresco<%classname%>", url = "${<%artifactId%>.url}", configuration = ClientConfiguration.class)
```

Comment:
Here, `vendorExtensions.x-service-url-prefix` is used as a workaround to avoid having beans from runtime bundle conflicting with the one from query service. However, the `x-service-url-prefix` is not compatible with bean names and requires extra changes on `patch.sh` to modify the name generated.